hw/ip/entropy_src/doc/checklist.md

```diff
@@ -35,25 +35,25 @@ Code Quality  | [LINT_SETUP][]                 | Done        |
 
 Type          | Item                    | Resolution  | Note/Collaterals
 --------------|-------------------------|-------------|------------------
-Documentation | [NEW_FEATURES][]        | Not Started |
-Documentation | [BLOCK_DIAGRAM][]       | Not Started |
-Documentation | [DOC_INTERFACE][]       | Not Started |
-Documentation | [MISSING_FUNC][]        | Not Started |
-Documentation | [FEATURE_FROZEN][]      | Not Started |
-RTL           | [FEATURE_COMPLETE][]    | Not Started |
-RTL           | [AREA_CHECK][]          | Not Started |
-RTL           | [PORT_FROZEN][]         | Not Started |
-RTL           | [ARCHITECTURE_FROZEN][] | Not Started |
+Documentation | [NEW_FEATURES][]        | Progressing |
+Documentation | [BLOCK_DIAGRAM][]       | Done        |
+Documentation | [DOC_INTERFACE][]       | Done        |
+Documentation | [MISSING_FUNC][]        | Progressing |
+Documentation | [FEATURE_FROZEN][]      | No          |
+RTL           | [FEATURE_COMPLETE][]    | Progressing |
+RTL           | [AREA_CHECK][]          | Done        |
+RTL           | [PORT_FROZEN][]         | Done        |
+RTL           | [ARCHITECTURE_FROZEN][] | Progressing |
```

Comment:
maybe also use `pending` as above?

Reply:
I think the word Progressing is appropriate since there are still open items still being discussed. Csrng is in a state of no major open items.

Reply:
@msfschaffner @tjaychen @eunchan I think this one can be closed quickly, just need feedback on how you want to handle the CDC setup I believe. Remember this is not a D2 close issue, but just a intermediate updated.